pkg/api/vm/handler.go

```diff
@@ -1428,6 +1428,11 @@ func getClonedVMYamlFromSourceVM(newVMName string, sourceVM *kubevirtv1.VirtualM
 			newVM.Annotations[cloneVMAnnoKey] = sourceAnnoValue
 		}
 	}
+
+	runStrategy := kubevirtv1.VirtualMachineRunStrategy(input.RunStrategy)
```

Comment:
How to ensure this statemment ?
```
Case 1: The valid value of runStrategy is emtpy, Always, Halted, Manual, RerunOnFailure, and Once. Other value will get error.
```

Reply:
@w13915984028 , kubevirt webhook handle for this part. Do we want to add related validation in API?

https://github.com/kubevirt/kubevirt/blob/f75b04afa5750369ea9f81a7693310db5c6313f6/pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go#L230-L264